README.md

```diff
@@ -1,3 +1,26 @@
+<p align="center">
+  <a href="https://www.pulumi.com?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=top-logo" title="Pulumi Examples - Build and Deploy Infrastructure as Code Solutions on Any Cloud">
+    <img src="https://www.pulumi.com/images/logo/logo-on-white-box.svg?" width="350">
+   </a>
+
+  [![Slack](http://www.pulumi.com/images/docs/badges/slack.svg)](https://slack.pulumi.com?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=slack-badge)
+  [![GitHub Discussions](https://img.shields.io/github/discussions/pulumi/pulumi)](https://github.com/pulumi/pulumi/discussions)
+  [![License](https://img.shields.io/github/license/pulumi/pulumi)](LICENSE)
+
+**Pulumi's Infrastructure as Code SDK** is the easiest way to build and deploy infrastructure, of any architecture and on any cloud, using programming languages that you already know and love. Code and ship infrastructure faster with your favorite languages and tools, and embed IaC anywhere with [Automation API](https://www.pulumi.com/docs/guides/automation-api/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=about-pulumi).
+
+Pulumi is open source under the [Apache 2.0 license](https://github.com/pulumi/pulumi/blob/master/LICENSE), supports many languages and clouds, and is easy to extend.  This repo contains the `pulumi` CLI, language SDKs, and core Pulumi engine, and individual libraries are in their own repos.
+
+## Table of contents
+
+- :rocket: [Pulumi Examples](#pulumi-examples)
```

Comment:
Looking at this list of items, I have a few questions:

* How are the first two different? The first one is the name of this repo -- so does that mean the second will take me somewhere else? If so, where to?
* Will the Community link take me somewhere else (like "to the Pulumi community")? If not, should it say something like "Pulumi Community resources", and maybe be sorted listed alongside Pulumi resources?
* Should we include a link to the FAQs in this TOC also, since it's a sibling of these other headings? 
* Should the contributing FAQ item be a heading of its own, since it's specific to this repo?
* Is Roadmap not a resource? What would I expect to see under Pulumi Resources (I'm actually not sure, until I click.)
* Should the repo-specific items be grouped together somehow, and the other items listed below them? For example, the first two links and the Contributors link are all related to this repo, but none of the other links are; they all seem to link off to destinations outside this repo. It makes the info feel less organized to me somehow.

Sorry for what may seem like nits -- just figured that if we're planning to propagate this overall structure to other repos as I think we are, we should probably tighten things up a bit.